tgui/packages/tgui/interfaces/LC13AbnormalityArchive.js

```diff
@@ -0,0 +1,130 @@
+import { sortBy } from 'common/collections';
+import { flow } from 'common/fp';
+import { useBackend, useLocalState } from '../backend';
+import { Box, Button, LabeledList, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { capitalize } from 'common/string';
+import { createLogger } from '../logging';
+import { LabeledListItem } from '../components/LabeledList';
+
+const logger = new createLogger();
+
+export const LC13AbnormalityArchive = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    abnormality_info,
+  } = data;
+  const abnormality_by_name = flow([
+    sortBy(abnormality_info => abnormality_info.name),
+  ])(data.abnormality_info || []);
+  const [
+    currentAbnormality,
+    setcurrentAbnormality,
+  ] = useLocalState(context, 'currentAbnormality', null);
+  return (
+    <Window
+      width={800}
+      height={600}>
+      <Window.Content>
+        <Stack fill>
+          <Stack.Item width="150px">
+            <Section fill scrollable>
+              {abnormality_by_name.map(f => (
+                <Button
+                  key={f.name}
+                  fluid
+                  color="transparent"
+                  selected={f === currentAbnormality}
```

Comment:
This is a strict equality, intended?

Reply:
yes?

Reply:
I mean the f is the selected abnormality shown on the button. You press the button and the current abnormality equals f. when the UI starts the currentAbnormality equals null.